### PR TITLE
Fix many background color and table pos.

### PR DESCRIPTION
**Pull request by shiipou**

### CODE REVIEW

**Comment by shiipou** [comment]
@pepa-linha 
I've made some change of the design to make it prettier on Chrome.
Check if it's ok for you

File: adminer.css [hunk 1]
```diff
@@ -363,11 +363,9 @@ p code + a:visited:hover {
 #tables {
     max-height: 100%;
     margin: 0 -15px !important;
-    position: absolute;
-    left: 30px;
-    right: 30px;
+    position: relative;
```
Comment (pepa-linha) [inline adminer.css:371]: Relative position is not possible here - when list contains many items - that is reason why `bottom: 0` and absolute position is here

File: adminer.css [hunk 2]
```diff
@@ -611,14 +609,18 @@ input.default {
 	font-weight: bold;
 }*/
 
+#help{
```
Comment (pepa-linha) [inline adminer.css]: When this element is showed, please? :)

File: adminer.css [hunk 3]
```diff
@@ -611,14 +609,18 @@ input.default {
...
...8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMy1jMDExIDY2LjE0NTY2MSwgMjAxMi8wMi8wNi0xNDo1NjoyNyAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENTNiAoTWFjaW50b3NoKSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDo5RUExMDNCMzUxRjYxMUU2ODAzQUNDMzMyNDgwMjFBRCIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDo5RUExMDNCNDUxRjYxMUU2ODAzQUNDMzMyNDgwMjFBRCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOjlFQTEwM0IxNTFGNjExRTY4MDNBQ0MzMzI0ODAyMUFEIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjlFQTEwM0IyNTFGNjExRTY4MDNBQ0MzMzI0ODAyMUFEIi8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+DQ1DzgAAAHlJREFUeNqs0NEGgDAYBeBTD7EY9RApddN9eu1uIrqOSHTVS/R3xh+REu3wse2M3wYRqSkhPHBd89J/EgJYKCOLe6x2M37GDRppooLMpTN6NumdXwncszQpxdTqvqKVBnjIdVBAOUW636gn8T3o/MpS1x3t8JRDgAEAnFdl2d2gogIAAAAASUVORK5CYII=') right center no-repeat;
```
Comment (pepa-linha) [inline adminer.css:650]: I do not like this change, sorry :-) Selects look like a text box with transparency color.

File: adminer.css [hunk 4]
```diff
@@ -355,6 +355,11 @@ p code + a:visited:hover {
     color: #fff;
 }
 
+.error b{
+    background-color: inherit;
+    font-weight: bold;
+}
+
```
Comment (pepa-linha) [inline adminer.css:362]: Thanks

File: adminer.css [hunk 5]
```diff
@@ -375,6 +375,26 @@ p code + a:visited:hover {
...
+    border-radius: 2px;
+    text-transform: uppercase;
+    letter-spacing: 0.5px;
+    line-height: 42px;
+    box-shadow: 0 3px 20px 0 rgba(93, 241, 220, 0.06);
+}
+
+#tables > li > a:nth-of-type(2):hover{
+    box-shadow: 0 3px 25px 1px rgba(93, 241, 220, 0.12);
+}
```
Comment (pepa-linha) [inline adminer.css]: View link is not button here.

File: README.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 [Adminer Design Dark]
 ===================================
 
-![Preview](https://raw.githubusercontent.com/pepa-linha/Adminer-Design-Dark/master/preview.png "Preview")
+![Preview](/preview.png "Preview")
```
Comment (pepa-linha) [inline README.md:4]: Thanks!

**Review by pepa-linha — CHANGES_REQUESTED** [review]
Hi, thank you for your PR. I made some changes.

**Comment by shiipou** [comment]
Ok, merge refused

Thanks for this great theme :)
Really love It

**Pull request closed without merging**